cmd/geoipupdate/defaults_notwin.go

```diff
@@ -0,0 +1,12 @@
+// +build !windows
+
+package main
+
+// These can be changed using compile time flags. e.g.,
+//
+// go build -ldflags "-X main.defaultConfigFile=/usr/local/etc/GeoIP.conf
+// -X main.defaultDatabaseDirectory=/usr/local/share/GeoIP"
```

Comment:
Maybe we could add a note to packagers in the README.md or somewhere.

Reply:
Yeah, good idea.